base-notebook/test/test_python.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+import logging
+
+from packaging import version
+
+LOGGER = logging.getLogger(__name__)
+
+
+def test_python_version(container, python_max_version="3.9"):
+    """Check that python version is not higher than a max version"""
```

Comment:
```suggestion
    """Check that python version is lower than a max version"""
```

I would also change the word max to the word next.

Reply:
Docstring changed, also change the word max to next as you suggested. I hope it will be Ok now.